README.md

```diff
@@ -5,7 +5,7 @@ Node.js
 
 Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. Node.js
 uses an event-driven, non-blocking I/O model that makes it lightweight and
-efficient. The Node.js package ecosystem, npm, is the largest ecosystem of open
+efficient. The Node.js package ecosystem, [npm](https://www.npmjs.com), is the largest ecosystem of open
```

Comment:
The link here in the original form of the PR was https://github.com/npm/npm (I suggested the current one), would you think that would be the better choice?